.github/workflows/storybook-chromatic.yml

```diff
@@ -144,6 +144,7 @@ jobs:
                   HOME: /root
                   # Update snapshots for PRs on the main repo, verify on forks, which don't have access to PostHog Bot
                   VARIANT: ${{ github.event.pull_request.head.repo.full_name == github.repository && 'update' || 'verify' }}
+                  STORYBOOK_SKIP_TAGS: 'test-skip,test-skip-${{ matrix.browser }}'
```

Comment:
Don't think we're using `test-skip-webkit` in this PR, is that intended?

Reply:
We are now. It was something I was going back and forth with to see if it fixes it or not. Seems to still only be failing on webkit consistently

Reply:
Oh to be clear - it is used in my toolbar PR. Thats the whole reason for this change :D